src/bulkAlloc.h

```diff
@@ -62,24 +62,20 @@
 
 #include "parser.h"
 
-/* creation and destroy */
+/*** serialized pool ***/
 SerializedPool *serPoolInit(RdbMemAlloc *mem);
 void serPoolRelease(RdbParser *p);
-
-/* allocation, flush & rollback */
-BulkInfo *serPoolAlloc(RdbParser *p, size_t len, AllocTypeRq typeRq, RdbBulk refBuf);
+BulkInfo *serPoolAlloc(RdbParser *p, size_t len, AllocTypeRq typeRq, char *refBuf);
 void serPoolFlush(RdbParser *p);
 void serPoolRollback(RdbParser *p);
-
-/* cloning RdbBulk */
-RdbBulkCopy serPoolCloneItem(RdbParser *p, BulkInfo *binfo);
-
-/* debug */
 void serPoolPrintDbg(RdbParser *p);
 int serPoolIsNewNextAllocDbg(RdbParser *p);
 
-/* Unmanaged */
-void allocUnmanagedBulk(RdbParser *p, size_t len, AllocUnmngTypeRq rq, char *refBuf, BulkInfo *bi);
-void freeUnmanagedBulk(RdbParser *p, BulkInfo *binfo);
+/*** Unmanaged allocation ***/
+void unmngAllocBulk(RdbParser *p, size_t len, AllocUnmngTypeRq rq, char *refBuf, BulkInfo *bi);
+void unmngFreeBulk(RdbParser *p, BulkInfo *binfo);
```

Comment:
i preferred the previous one, "unmng" is not a thing.

Reply:
I was looking for a common prefix that is aligned with the header file description and API and makes clear distinction  between serialized-pool bulk management vs. unmanaged bulks ( `serPoolXXX` vs `unmngXXX`). 

For this case, I personally prefer clartiy over grammar correctness.

Reply:
so maybe they should all have a `bulkAlloc` prefix, and it should be `bulkAllocUnmanagedXXXX` and `bulkAllocSerializedPoolXXX`.
or something shorter but still using complete words or common terms.
i.e. `allocPoolXXXX` and `allocUnmanagedXXX` or alike.
Maybe "unmanaged" can be replaced with "user"?

Reply:
fine.. if it makes sense to you (i'm not following this closely at the moment, too busy).
if we can avoid the `unmng` abbreviation i'd prefer it. and if we can have a common prefix for both allocator that's also preferred.